core/src/main/java/com/linecorp/armeria/common/MediaType.java

```diff
@@ -856,4 +935,59 @@ private static String escapeAndQuote(String value) {
         }
         return escaped.append('"').toString();
     }
+
+    /**
+     * Returns {@code true} if {@code actualParameters} contains all entries of {@code expectedParameters}.
+     * Note that this method does *not* require {@code actualParameters} to contain *only* the entries of
```

Comment:
I'm just curious that `*not*` and `*only*` are for the emphasis? because I've never seen that before......